package.json

```diff
@@ -24,8 +24,8 @@
   },
   "dependencies": {
     "cheerio": "1.0.0-rc.2",
-    "css-tree": "1.0.0-alpha.26",
-    "csso": "3.4.0",
+    "css-tree": "1.0.0-alpha.27",
+    "csso": "~3.5.0",
```

Comment:
Because we pass an AST to CSSO, so used CSSTree version should be the same (or greater if that version has no critical changes) as used by CSSO. Currently, CSSO can bump CSSTree version in its minor version, as noted in [readme](https://github.com/css/csso/blob/master/README.md):
> Warning: CSSO uses early versions of CSSTree that still in active development. CSSO doesn't guarantee API behind `syntax` field or AST format will not change in future releases of CSSO, since it's subject to change in CSSTree. Be careful with CSSO updates if you use `syntax` API until this warning removal.